### DISCUSSION TITLE
CLI arguments

### DISCUSSION THREAD

**Discussion by sarmong** [elkowar/eww#642] [Q&A]
Can I pass CLI argument to "eww open" command?

**Comment by sarmong** [comment]
Seems like not.

Can I at least read from a file?

I want to create a simple yes/no prompt that I can use to call custom commands on yes and no.

**Reply by mantacid** [reply]
The only arguments seem to be basic stuff like window dimensions and position, perhaps some others. Run `eww open —help` for the rest.

There’s probably a really Hacky way to get around the limitation of not being able to pass arguments to windows. My idea is to use a script to handle both the creation of some yuck that stores the arguments and the window itself, which then`include`s these variables

**Comment by elkowar** [comment]
Will be possible soon as soon as we can merge #431!
